_posts/en/posts/2018-09-20-notice.md

```diff
@@ -69,6 +71,7 @@ September 19, 2018:
 
 September 20, 2018: 
 
-- 19:50 A developer by the title earlz independently discovered and reported the vulnerability to the Bitcoin Core security contact email.
+- 16:21 This full disclosure notice was posted to bitcoincore.org
```

Comment:
This notice wasn't reported to BitcoinCore.org until Sept 21 at roughly 00:30 UTC.